README.md

```diff
@@ -52,7 +52,9 @@ copy of the cvmfs software.  The cvmfs software and configuration are
 expected to be in a `dist` subdirectory under where the scripts are.  The
 easiest way to create the dist directory is to use `makedist`.  It takes a
 parameter of `osg`, `egi`, or `default` to download the latest cvmfs and
-configuration rpm from one of those three sources.
+configuration rpm from one of those three sources. Note: `egi` does not
+currently provide rpms for RHEL8/9, and `osg` does not currently provide
+rpms for RHEL9.
```

Comment:
I found out that OSG does have quite a few rpms in an [el9 development repo](https://repo.opensciencegrid.org/osg/3.6/el9/development/x86_64/).  I will think about putting in a temporary hack for OSG to use that.